### PR TITLE
Refactor end game win game. Closes #89

### LINKED ISSUE #89: Refactor endGame winGame functions

### PR DESCRIPTION
**Pull request by kbs5280**
@neight-allen @twhitinger 

This PR extracts the logic from the endGame and winGame functions to the setFinalScreen function. The reduces duplication of code and improves readability.

Please review the following functions: `endGame()`, `winGame` and `setFinalScreen`.

If you would like to manually test this, you can clone the repo, cd into the repo, run $npm install, run $ npm start, visit http://localhost:8080/webpack-dev-server/, and play the game until you win and/or lose. All scores should be visible.

@twhitinger, please review the refactor and comment on the changes.

### CODE REVIEW

**Comment by twhitinger** [comment]
It looks like you are able to reuse the same setFinalScreen function for multiple screens you are displaying. Great job at drying up your code. additional you call multiple functions within the endgame() and don't have any loose code so it looks like you went through multiple refactors prior to organize.

**Comment by kbs5280** [comment]
@twhitinger Thanks for the comments. We will keep that in mind as we move forward. @matthewrpacker I think we are ready to merge this refactor, please review.

**Comment by matthewrpacker** [comment]
I like how you separated the logic from endGame and winGame.  Nice refactor.  This looks ready to merge.  I will handle the merge.

**Review by neight-allen — COMMENTED** [review]
I remember looking at this refactor. It's a really good first step. I wonder if you can eliminate even more of those parameters from the `setFinalScreen` function.

File: lib/game.js [hunk 1]
```diff
@@ -29,16 +29,21 @@ star.src = "star.png";
...
-let bottomObstacle = new Obstacle({x: canvas.width, y: canvas.height-200, width: 50, height: 50});
-let topObstacle_2 = new Obstacle({x: canvas.width+600, y: 200, width: 50, height: 50});
-let bottomObstacle_2 = new Obstacle({x: canvas.width+600, y: canvas.height-300, width: 50, height: 50});
-let topObstacle_3 = new Obstacle({x: canvas.width+600, y: 0, width: 50, height: 50});
-let bottomObstacle_3 = new Obstacle({x: canvas.width+600, y: canvas.height-75, width: 50, height: 50});
-let topObstacle_4 = new Obstacle({x: canvas.width, y: 0, width: 50, height: 50});
-let bottomObstacle_4 = new Obstacle({x: canvas.width, y: canvas.height-75, width: 50, height: 50});
-let topObstacle_5 = new Obstacle({x: canvas.width+600, y: 0, width: 50, height: 50});
-let bottomObstacle_5 = new Obstacle({x: canvas.width+600, y: canvas.height-75, width: 50, height: 50});
+const obstacleWidth = 50
```
Comment (neight-allen) [inline lib/game.js:32]: There's a bunch of stuff in here that isn't relevant to your refactor. Try to isolate changes in your commits.

**Pull request merged**